### PR TITLE
tidb: doc for multi-valued index GA

### PR DESCRIPTION
**Pull request by xiongjiwei**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v7.0 (TiDB 7.0 versions)
- [ ] v6.6 (TiDB 6.6 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.4 (TiDB 6.4 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Review by qw4990 — APPROVED** [review]
LGTM

**Review by ran-huang — APPROVED** [review]

File: sql-statements/sql-statement-create-index.md [hunk 1]
```diff
@@ -375,13 +371,14 @@ Query OK, 1 row affected (0.00 sec)
...
 - 如果是空 JSON 数组，则不会有对应的索引记录。
-- `CAST(... AS ... ARRAY)` 中的目标类型不能是 `BINARY`、`JSON`、`YEAR`、`FLOAT`、`DOUBLE`、`DECIMAL`。其中源类型必须是 JSON。
+- `CAST(... AS ... ARRAY)` 中的目标类型不能是 `BINARY`、`JSON`、`YEAR`、`FLOAT`、`DECIMAL`。其中源类型必须是 JSON。
 - 无法使用多值索引进行排序。
 - 只允许在 JSON 数组上建立多值索引。
 - 多值索引不可以作为主键或外键。
 - 多值索引使用额外的存储空间为：平均每行数组元素个数 * 普通二级索引使用空间。
 - 相比于普通索引，DML 会对多值索引产生更多的索引记录的修改，因此多值索引会带来比普通索引更大的性能影响。
 - 由于多值索引是一种特殊的表达式索引，因此具有表达式索引的限制。
+- 使用备份恢复工具（BR），同步工具（TiCDC），导入工具（Lightning）无法将定义了多值索引的表备份/同步/导入到低于 v6.6.0 版本的 TiDB。
```
Comment (ran-huang) [inline sql-statements/sql-statement-create-index.md:381]:
```suggestion
- 使用备份恢复工具 (BR)，同步工具 (TiCDC)，导入工具 (TiDB Lightning) 无法将定义了多值索引的表备份、同步、导入到低于 v6.6.0 版本的 TiDB。
```

File: sql-statements/sql-statement-create-index.md [hunk 2]
```diff
@@ -375,13 +371,14 @@ Query OK, 1 row affected (0.00 sec)
...
 - 如果是空 JSON 数组，则不会有对应的索引记录。
-- `CAST(... AS ... ARRAY)` 中的目标类型不能是 `BINARY`、`JSON`、`YEAR`、`FLOAT`、`DOUBLE`、`DECIMAL`。其中源类型必须是 JSON。
+- `CAST(... AS ... ARRAY)` 中的目标类型不能是 `BINARY`、`JSON`、`YEAR`、`FLOAT`、`DECIMAL`。其中源类型必须是 JSON。
 - 无法使用多值索引进行排序。
 - 只允许在 JSON 数组上建立多值索引。
 - 多值索引不可以作为主键或外键。
 - 多值索引使用额外的存储空间为：平均每行数组元素个数 * 普通二级索引使用空间。
 - 相比于普通索引，DML 会对多值索引产生更多的索引记录的修改，因此多值索引会带来比普通索引更大的性能影响。
 - 由于多值索引是一种特殊的表达式索引，因此具有表达式索引的限制。
+- 使用备份恢复工具 (BR)，同步工具 (TiCDC)，导入工具 (TiDB Lightning) 无法将定义了多值索引的表备份、同步、导入到低于 v6.6.0 版本的 TiDB。
```
Comment (qiancai) [inline sql-statements/sql-statement-create-index.md:381]:
```suggestion
- 使用备份恢复工具 (BR)、同步工具 (TiCDC)、导入工具 (TiDB Lightning) 无法将定义了多值索引的表备份、同步、导入到低于 v6.6.0 版本的 TiDB。
```

**Review by qiancai — APPROVED** [review]
other LGTM

**Pull request merged**